src/lib/helpers/api/get-mod-database.ts

```diff
@@ -3,6 +3,18 @@ import { readFromStore } from '../read-from-store';
 
 const DATABASE_URL = 'https://raw.githubusercontent.com/ow-mods/ow-mod-db/master/database.json';
 
+// Caution: must be synced with tags in ow-mod-db repo.
```

Comment:
Could the possible values be generated by the values found in the fetched database to avoid this?

Reply:
they already are. But this is for type safety. The only way to centralize it would be to publish something to npm that contains the type definitions and common methods between website and database.

Reply:
That said, I may not need type safety if the code on the website doesn't need to be aware of specific tags (I started off that way and then changed it). After I'm done implementing this, I might clean it up and just store it as plain string.